app/src/main/java/edu/uc/kansakpb/ikeahjalpare3/dao/IFurnitureDAO.kt

```diff
@@ -3,9 +3,15 @@ package edu.uc.kansakpb.ikeahjalpare3.dao
 import edu.uc.kansakpb.ikeahjalpare3.dto.Furniture
 import retrofit2.http.GET
 import retrofit2.Call
+import retrofit2.http.Query
 
 interface IFurnitureDAO {
-    //Created DAO class for
+
     @GET("/ikeafurniture.json")
     fun getAllFurnitures(): Call<ArrayList<Furniture>>
+
+    @GET("/ikeafurniture.json")
+    fun getFurniture(@Query("furniture_name")funritureName: String): Call <ArrayList<Furniture>>
+
```

Comment:
Is this needed?  Does the JSON endpoint accept a parameter named furniture_name?  Is this called from anywhere else in code?  If not, this is clutter and should not be merged.